XTS/WS-C/dev/src/com/arjuna/wsc11/messaging/ActivationCoordinatorProcessorImpl.java

```diff
@@ -65,7 +65,7 @@
                 try
                 {
                     final Expires expiresElement = createCoordinationContext.getExpires() ;
-                    final Long expires = (expiresElement == null ? null : new Long(expiresElement.getValue())) ;
+                    final Long expires = (expiresElement == null ? null : Long.valueOf(expiresElement.getValue())) ;
```

Comment:
Unnecessary boxing, `expiresElement.getValue()` is sufficient. Similarly elsewhere in this PR.